src/js/utils/DOM.js

```diff
@@ -64,18 +64,38 @@ export const containsFocus = node => {
   return !!element;
 };
 
-export const getFirstFocusableDescendant = element => {
+/**
+ * Check if the element.tagName is an input, select or textarea
+ * @param {Element} element
+ * @returns boolean
+ */
```

Comment:
```suggestion
// Check if the element.tagName is an input, select or textarea
```
I think we can just have this as a normal comment since we don't use JSDoc style comments anywhere else